src/pal/src/misc/miscpalapi.cpp

```diff
@@ -298,3 +301,153 @@ PAL_Random(
     PERF_EXIT(PAL_Random);
     return bRet;
 }
+
+
+// Support for EnsureOpenSslInitialized
+
+static const char * const libcryptoName = "libcrypto" PAL_SHLIB_SUFFIX;
```

Comment:
Fun :smile: Thanks for the info, and for thinking about it.  I'd like to go ahead with the change as is, and open an issue to figure out what to do about Linux "dll hell" ;)  This will affect not only this particular function, but more broadly DllImports from managed code.  Sound good?